pennylane/wires.py

```diff
@@ -34,15 +34,19 @@ def _process(wires):
         # interpret as a single wire
         return (wires,)
 
+    elif getattr(wires, "shape", None) == tuple():
```

Comment:
I suppose the key here, is that if `wires` does not have a `shape` attribute, we don't want this `elif` statement to evaluate as `True`. So the default must be anything that causes the equality to fail; it can be anything _except_ `tuple()`.

Originally, I had this more explicit:
```python
elif hasattr(wires, "shape") and wires.shape == tuple():
```
I reduced it to the above because I was like 'shorter is more elegant'. But, perhaps better to make it explicit!

Reply:
```suggestion
    elif hasattr(wires, "shape") and wires.shape == tuple():
```